environment.yml

```
@@ -1,304 +1,31 @@
-  - pixman=0.44.2
-  - platformdirs=4.3.6
-  - pox=0.3.5
-  - ppft=1.7.6.9
-  - progressbar2=4.5.0
-  - proj=9.5.1
-  - prompt-toolkit=3.0.50
-  - psutil=6.1.1
-  - psycopg2=2.9.9
-  - pthread-stubs=0.4
-  - ptyprocess=0.7.0
-  - pure_eval=0.2.3
-  - pycparser=2.22
-  - pygments=2.19.1
-  - pyparsing=3.2.1
-  - pyrosar=0.28.0
-  - pyside6=6.8.1
-  - pysocks=1.7.1
-  - python=3.10.16
-  - python-dateutil=2.9.0.post0
-  - python-tzdata=2025.2
-  - python-utils=3.9.1
-  - python_abi=3.10
-  - pytz=2024.2
-  - pyyaml=6.0.2
-  - pyzmq=26.2.1
-  - qhull=2020.2
-  - qt6-main=6.8.1
-  - rasterio=1.4.3
-  - rav1e=0.6.6
-  - readline=8.2
-  - requests=2.32.3
-  - s1etad=0.5.3
-  - s1etad_tools=0.8.1
-  - scipy=1.15.2
-  - setuptools=75.6.0
-  - setuptools-scm=8.1.0
-  - setuptools_scm=8.1.0
-  - shapely=2.0.6
-  - simplekml=1.3.6
-  - six=1.17.0
-  - snappy=1.2.1
-  - snuggs=1.4.7
-  - spatialist=0.14.0
-  - sqlalchemy=1.4.54
-  - sqlalchemy-utils=0.41.2
-  - sqlalchemy-utils-arrow=0.41.2
-  - sqlalchemy-utils-babel=0.41.2
-  - sqlalchemy-utils-base=0.41.2
-  - sqlalchemy-utils-color=0.41.2
-  - sqlalchemy-utils-encrypted=0.41.2
-  - sqlalchemy-utils-intervals=0.41.2
-  - sqlalchemy-utils-password=0.41.2
-  - sqlalchemy-utils-pendulum=0.41.2
-  - sqlalchemy-utils-phone=0.41.2
-  - sqlalchemy-utils-timezone=0.41.2
-  - sqlalchemy-utils-url=0.41.2
-  - sqlite=3.47.2
-  - stack_data=0.6.3
-  - svt-av1=2.3.0
-  - tblib=3.0.0
-  - time-machine=2.16.0
-  - tk=8.6.13
-  - tomli=2.2.1
-  - tornado=6.4.2
-  - traitlets=5.14.3
-  - types-python-dateutil=2.9.0.20241206
-  - typing-extensions=4.12.2
-  - typing_extensions=4.12.2
-  - tzdata=2024b
-  - unicodedata2=16.0.0
-  - uriparser=0.9.8
-  - urllib3=2.3.0
-  - wayland=1.23.1
-  - wcwidth=0.2.13
-  - wheel=0.45.1
-  - x265=3.5
-  - xcb-util=0.4.1
-  - xcb-util-cursor=0.1.5
-  - xcb-util-image=0.4.0
-  - xcb-util-keysyms=0.4.1
-  - xcb-util-renderutil=0.3.10
-  - xcb-util-wm=0.4.2
-  - xerces-c=3.2.5
-  - xkeyboard-config=2.43
-  - xorg-libice=1.1.2
-  - xorg-libsm=1.2.6
-  - xorg-libx11=1.8.12
-  - xorg-libxau=1.0.12
-  - xorg-libxcomposite=0.4.6
-  - xorg-libxcursor=1.2.3
-  - xorg-libxdamage=1.1.6
-  - xorg-libxdmcp=1.1.5
-  - xorg-libxext=1.3.6
-  - xorg-libxfixes=6.0.1
-  - xorg-libxi=1.8.2
-  - xorg-libxrandr=1.5.4
-  - xorg-libxrender=0.9.12
-  - xorg-libxtst=1.2.5
-  - xorg-libxxf86vm=1.1.6
-  - yaml=0.2.5
-  - zeromq=4.3.5
-  - zipp=3.21.0
-  - zlib=1.3.1
-  - zstandard=0.23.0
-  - zstd=1.5.6
-  - pip:
-    - asf_search==8.1.1
-    - black==25.1.0
-    - boto3==1.36.4
-    - botocore==1.36.4
-    - geopandas==1.0.1
-    - iniconfig==2.0.0
-    - jmespath==1.0.1
-    - pluggy==1.5.0
-    - pyogrio==0.10.0
-    - pyproj==3.7.0
-    - pytest==8.3.4
-    - s3transfer==0.11.1
-    - tzdata==2024.2
+- affine >=2.4.0
+- click >=8.1.8
+- geopandas >=1.0.1
+- numpy <2.0
+- gdal >=3.10.2
+- pyproj >=3.7.1
+- pyrosar >=0.28.0
+- rasterio >=1.4.3
+- shapely >=2.0.7
+- spatialist >=0.14.0
+- s1etad >=0.5.5,<0.6
```

```suggestion
- s1etad >=0.5.5
```
I forgot to remove the upper bound when I installed